### PR TITLE
l20n.js

### PR DESCRIPTION
**Pull request by dkackman**
Should address https://github.com/Chia-Network/website/issues/3

Includes 
- l20n.js
- full translation attributes for index.html
- english, japanese and french (partial) translation files
- update to readme to explain how to add translations

If accepted these PRs will need to be done differently:
https://github.com/Chia-Network/website/pull/9
https://github.com/Chia-Network/website/pull/5 (needs to be checked as I manually moved the kanji)
https://github.com/Chia-Network/website/pull/7

### CODE REVIEW

File: webroot/locales/app.ja.ftl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+community = コミュニティ
+
+about-us = 私達について
+
+jobs = 求人
+
+learn-more = 更に詳しく知る
+
+faq = Frequently Asked Questions
```
Comment (katabame) [inline webroot/locales/app.ja.ftl]: In Japanese, `Frequently Asked Questions` will be translated to `よくある質問`.

File: webroot/locales/app.ja.ftl [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// for ease of re-use put short headings, section titles and keywords up here
+
+hiring = 人材募集中！
+
+get-chia = 得る Chia
```
Comment (katabame) [inline webroot/locales/app.ja.ftl]: `得る Chia` is not good Japanese, un translate this line(replace to `Get Chia`) is better.

**Comment by richardkiss** [comment]
There are a couple simple merge issues, can you please resolve?

Can you include an example in index.html that lets the user override their language manually?

**Comment by dkackman** [comment]
conflicts resolved.

> Can you include an example in index.html that lets the user override their language manually?

Not sure I understand what you're asking for. Are you looking for a language selector drop down in index.html? l20n uses the browser's language preference by default

**Comment by dkackman** [comment]
I added language selection links in the footer. To maintain the manually selected language, I set a cookie so took a dependency on jquery and js-cookie. 

I also have a changeset queued to manage the faq translation resolution which, since those are in markdown won't need l20n.

**Comment by richardkiss** [comment]
Very nice, thank you!!

**Comment by dkackman** [comment]
@brampauwelyn take a look at this pr for adding faq translations https://github.com/Chia-Network/website/pull/13

**Pull request merged**